wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewController.swift

```diff
@@ -108,17 +108,12 @@ final class ConversationGuestOptionsViewController: UIViewController,
         tableView.dataSource = self
         tableView.backgroundColor = SemanticColors.View.backgroundDefault
         tableView.contentInsetAdjustmentBehavior = .never
-        setupNavigationBar()
     }
 
     private func setupNavigationBar() {
         navigationController?.navigationBar.tintColor = SemanticColors.Label.textDefault
-        if var textAttributes = navigationController?.navigationBar.titleTextAttributes {
-            textAttributes[NSAttributedString.Key.foregroundColor] = SemanticColors.Label.textDefault
-            navigationController?.navigationBar.titleTextAttributes = textAttributes
-        }
 
-        navigationItem.setupNavigationBarTitle(title: L10n.Localizable.GroupDetails.GuestOptionsCell.title.capitalized)
+        setupNavigationBarTitle(with: L10n.Localizable.GroupDetails.GuestOptionsCell.title.capitalized)
```

Comment:
I think our copywriter Antje told me once, that she is not happy with the code base capitalisation, where they are out of control. Maybe talk to her to remove it now when touching all of these titles anyway?

Example `Register a Device` would become `Register A Device`

Reply:
Why do we even do capitalization transformations? I don't think they're locale aware, so what works in one language may not for others. In my opinion, we should leave the strings as they are translated (capitalization included).

Reply:
@markusfassbender Done: https://github.com/wireapp/wire-ios/commit/8b7d5c23e08a87f47ebaaae421af721e9231d30c